API.md

```diff
@@ -0,0 +1,55 @@
+# Octofet API
+
+## `class Octofet`
+
+Create an object of type `Octofet` to communicate with a particular [Octofet board](https://my.amperka.com/modules/octofet). One Octofet board provides 8 power switches also known as “channels”.
+
+### `Octofet(uint8_t pinCS, uint8_t deviceCount = 1)`
+
+Constructs a new Octofet board object that uses the default hardware SPI bus.
+
+- `pinCS`: the `CS` pin used to control the shift-register latch. It is marked `CS` on the Octofet board. Any GPIO pin of the controller may be used.
```

Comment:
```suggestion
- `pinCS`: the chip select (also known as slave select) pin used to control the shift-register latch. It is marked `CS` on the Octofet board. Any GPIO pin of the controller may be used.
```

More informative

Reply:
Slave Select usually called `SS`.

Reply:
We have a wrongly named pin on the board. It is called CS. The action that it performs in the system is not a "chip select" at all. I propose to pronounce it as "Communication Start." I think this is more consistent with the actions performed by him.

Reply:
These are details. The main idea is to match arg names to labels on the board and the traditional names of SPI... OK, we can write “the chip select (also known as SS or Slave Select) pin” but don’t overwhelm a reader more than that. Perhaps, it would be better to just drop this note:

“the chip select pin” ← period

BTW, what’s wrong with the Chip Select term? Why should it be Communication Start? I see the latter for the first time.

Reply:
`what’s wrong with the Chip Select term?` - we select not one chip but chain of chips.

Reply:
I’m against synthetic new terms not seen in other places. CS is widely known as a part of SPI concept. The fact it potentially selects multiple chips depends on the point of view. You can think of the daisy-chain as of a single device with 16, 24, etc channels. Hope this makes sense.

Reply:
Ok, used option from @nkrkv